data/ai/micro_ais/scenarios/dungeon_mai_comparisons.cfg

```diff
@@ -0,0 +1,294 @@
+            description= _ "End Scenario"
+            image=items/ring-red.png~CROP(26,26,20,20)
+            [show_if]
+                {VARIABLE_CONDITIONAL scenario_name equals boss_mai_comparisons}
+            [/show_if]
+            [command]
+                [message]
+                    speaker=narrator
+                    image=wesnoth-icon.png
+                    message= _ "Well, that was that."
+                [/message]
+
+                # So that game goes on to next scenario
+                [modify_side]
+                    side=5
+                    controller=human
+                [/modify_side]
+
+                [endlevel]
+                    result=victory
+                    bonus=no
+                    carryover_percentage=0
+                    carryover_report=no
+                    linger_mode=no
+                [/endlevel]
+            [/command]
+        [/set_menu_item]
+        [set_menu_item]
+            id=m02_new_unit_ruffian
+            description= _ "Place Side 4 Ruffian"
+            image=units/human-outlaws/ruffian.png~CROP(23,18,24,24)
+            [show_if]
+                {VARIABLE_CONDITIONAL scenario_name equals boss_mai_comparisons}
+            [/show_if]
+            [command]
+                {GENERIC_UNIT 4 Ruffian $x1 $y1}
+            [/command]
+        [/set_menu_item]
+        [set_menu_item]
+            id=m02_new_unit_footpad
+            description= _ "Place Side 4 Footpad"
+            image=units/human-outlaws/footpad.png~CROP(23,18,24,24)
+            [show_if]
+                {VARIABLE_CONDITIONAL scenario_name equals boss_mai_comparisons}
+            [/show_if]
+            [command]
+                {GENERIC_UNIT 4 Footpad $x1 $y1}
+            [/command]
+        [/set_menu_item]
+    [/event]
+
+    [event]
+        name=start
+
+        [message]
+            speaker=narrator
+            image=wesnoth-icon.png
+            message= _ "Hello, and welcome to the Dungeon where the boss micro AI can be probed and investigated."
+        [/message]
+
+        [message]
+            speaker=narrator
+            image=wesnoth-icon.png
+            message= _ "The boss micro AI has been assigned to a boss-class unit along with 2 other identical units which have the Zone Guardian Micro AI and the Hunter MAI respectively. This was done so was to investigate any behavioural changes and similaries can be observed."
+        [/message]
+
+        [message]
+            speaker=narrator
+            image=wesnoth-icon.png
+            message= _ "The Boss Micro AI will initially wander around in its arena. However, if an enemy unit is within its zone, it will approach and target the unit. On kill, the boss unit will check for any nearby targets. If any are found, it will attack those. If not, he will return to wandering around in its zone. If an enemy unit decides to run away, the boss unit will pursue and will not stop until the enemy has been dealt with."
```

Comment:
There's a pronoun mismatch in there ("he" as opposed to "it"). Trivial stuff, but might as well comment on it.

Reply:
fixed.